common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -280,6 +321,7 @@ def load_item(self, location):
 
                 # decache any computed pending field settings
                 module.save()
+                self.cached_modules[location] = module
```

Comment:
Here, is `location` something that has the revision in it, or has it been stripped. If it's been stripped, is that a problem?